### DISCUSSION TITLE
Which 🇩🇪 "Thorsten" Tacotron2 DDC model sounds better to be released?

### DISCUSSION THREAD

**Discussion by thorstenMueller** [coqui-ai/TTS#1875] [Versions & Releases]
I've trained a new German TTS model (Tacotron2 DDC) and am unsure which checkpoint is better to be released. If you like you can rate which version sounds better to you.

You can hear samples and vote here. The more poular model will be the one to be released in one of next 🐸 TTS releases.
https://www.thorsten-voice.de/2022/08/14/welches-tts-modell-klingt-besser/

**Comment by Ca-ressemble-a-du-fake** [comment]
Very good sound quality ! Both models sound so natural. FYI in the first paragraph (variante 1) the last word "schien" is missing.

**Reply by thorstenMueller** [reply]
> Very good sound quality ! Both models sound so natural. FYI in the first paragraph (variante 1) the last word "schien" is missing.

Thanks for your nice feedback. Yes, i've heard this too. But as variation 1 is speaking a little bit slower the end of the phrase was abrupted by max_decoder_steps. But this shouldn't be a tough problem.
